Sources/NIOHTTP2/HTTP2FrameParser.swift

```diff
@@ -779,8 +781,9 @@ struct HTTP2FrameDecoder {
         var consumed = 0
         while consumed < length {
             // TODO: name here should be HTTP2SettingsParameter(fromNetwork:), but that's currently defined for NGHTTP2's Int32 value
-            let identifier = HTTP2SettingsParameter(fromPayload: bytes.readInteger()!)
-            let value: UInt32 = bytes.readInteger()!
+            let values = bytes.readMultipleIntegers(as: (UInt16, UInt32).self)!
```

Comment:
This can be cleaned up as:

```suggestion
            let (rawIdentifier, value) = bytes.readMultipleIntegers(as: (UInt16, UInt32).self)!
```